### PR TITLE
Twitter account by languages

### LINKED ISSUE #457: Bouton de partage Twitter utilise le compte_EN sur la version FR du site
Le bouton de partage Twitter sur la version /fr/ du site fait mention du profil https://twitter.com/cafebabel_ENG , alors qu'il faudrait logiquement qu'il mentionne https://twitter.com/cafebabel_FR

### PR DESCRIPTION
**Pull request by johangiraud**
fix #457

### CODE REVIEW

**Comment by johangiraud** [comment]
I am stuck with this error: `TypeError: 'str' object is not callable`. Can you help @davidbgk @vinyll ?

**Comment by johangiraud** [comment]
Here I just need help to fix the test, otherwise I am good @vinyll

File: cafebabel/templates/articles/detail.html [hunk 1]
```diff
@@ -105,7 +105,14 @@ <h1>{{ article.title }}</h1>
   {% block share_nav %}
   <nav class=share>
   <p>Loved this story? Then tell your friends:
-    <a href="https://twitter.com/share?url={{ request.url|quote_plus }}&text={{ article.title }}&via=cafebabel_eng">Twitter</a>
+    {% set lang = current_language %}
+    {% if lang == 'en' %}{% set twitter_nickname = 'cafebabel_ENG' %}{% endif %}
```
Comment (vinyll) [inline cafebabel/templates/articles/detail.html]: Can you keep "eng" lowercase as that breaks the tests?
If not the test need to be adapted to have "ENG" uppercase ;)

**Review by vinyll — CHANGES_REQUESTED** [review]

File: cafebabel/templates/articles/detail.html (hunk 1 above)
Comment (johangiraud) [reply cafebabel/templates/articles/detail.html]: ha ! sure ! I thought we need to keep the right case ! good https://github.com/cafebabel/cafebabel.com/pull/486/commits/6659587ebae852f988c107fbd6be6c1ffbc56c9c

**Review by vinyll — APPROVED** [review]
👌

**Pull request merged**